webhook_to_fedora_messaging/endpoints/message.py

```diff
@@ -0,0 +1,36 @@
+from flask import Blueprint, Flask, request, Response, Request
+from ..database import db
+from ..models.service import Service
+from ..models.user import User
+from sqlalchemy_helpers import get_or_create
+from .util import not_found, success, bad_request, created, conflict, validate_request, create_event, query_fasjson
+from .parser.parser import msg_parser
+from fedora_messaging import api
+
+
+app = Flask(__name__)
+message_endpoint = Blueprint("message_endpoint", __name__)
+
+
+@message_endpoint.route("/message", methods=["POST"])
+def create_msg():
+    """
+    Used for creating a new message by sending a post request to /message path
+        
+        Request Body:
+        service_uuid: Service related to message.    
+        
+    """
+    if not validate_request(request, fields=['service_uuid']):
+        return bad_request()
+    
+    session = db.Session()
+    body = request.json
+    
+    service = session.query(Service).filter(Service.id == body['service_uuid']).first()
+    
+    if service is None:
+        return not_found()
+    msg = msg_parser(service.type, request)
```

Comment:
Also, if as suggested below this function can raise `ValueError` on unknown service types, this should be caught here to return something appropriate (a 4xx response type).